tools/build_config/build.sh

```diff
@@ -85,6 +85,41 @@ function run_package() {
   echo_and_run "cpack -G TGZ"
 }
 
+function build_docs() {
+    # Update release numbers
+    build_id=$(sed -nr '/CPACK_PACKAGE_FILE_NAME/{s/.*"Horace-([^"]+)".*/\\1/p};' ./build/CPackConfig.cmake)
```

Comment:
Is `build_id` the right name for this variable? Aren't you pulling the version here?

Reply:
Depends. If it is the nightly, I think it's the SHA/date, which isn't the version?